tests/datasource/fluent/test_viral_snippets.py

```diff
@@ -29,6 +29,7 @@
 logger = logging.getLogger(__file__)
 
 
+@pytest.mark.xfail(reason="YAML file shares state with other tests")
```

Comment:
I don't love this but I'd ask to come back to this once I wrap up the `save` work - we add ids from another test I believe and then hit an assertion error here because we don't expect those values

Reply:
We discussed a path forward here.